lib/src/test/java/io/ably/lib/test/rest/RestAuthTest.java

```diff
@@ -106,6 +106,7 @@ public void authinit2() {
 			opts.port = testVars.port;
 			opts.tlsPort = testVars.tlsPort;
 			opts.tls = testVars.tls;
+			opts.httpMaxRetryCount = testVars.httpMaxRetryCount;
```

Comment:
All of these httpMaxRetryCount configurations are still here and not related to the test in hand at all?  Why can't they simply use the default?